frontend/src/layout/navigation/SideBar/SideBar.scss

```diff
@@ -102,6 +102,10 @@
     max-height: calc(100vh - 20rem);
     max-width: calc(100vw - 14rem);
 
+    &[data-attr='sidebar-new-insights'] {
+        max-width: 21rem;
+    }
```

Comment:
This style would also be useful on the "New insight" popover on the insights list page. Can we put it on `.new-insight-overlay` instead of this `SideBar` override?

Reply:
Hmmm, interesting, that already has an override that's being ignored

<img width="883" alt="Screenshot 2023-03-16 at 11 30 01" src="https://user-images.githubusercontent.com/984817/225603836-270620e9-60a5-4937-acc6-924935d17317.png">

*rolls up sleeves

Reply:
How about passing optionally passing max width to popover....

<img width="349" alt="Screenshot 2023-03-16 at 11 40 08" src="https://user-images.githubusercontent.com/984817/225606328-c230e80b-0084-4952-b190-cf3eb8cbc69a.png">
<img width="438" alt="Screenshot 2023-03-16 at 11 39 16" src="https://user-images.githubusercontent.com/984817/225606333-dad2cc29-4b09-4b23-986c-3b2b8a08c49b.png">
<img width="550" alt="Screenshot 2023-03-16 at 11 39 09" src="https://user-images.githubusercontent.com/984817/225606336-d812dcdb-284f-4473-8534-886ac19cb8a2.png">